stash-cache-origin/supervisord_startup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+/usr/sbin/setup_xrootd_certs.sh
+
+# Give the chance to the pod to initialize host-specific info
+source /usr/sbin/pod_init.sh
+
+# Now we can actually start the supervisor
+exec /usr/bin/supervisord -c /etc/supervisord.conf
+
```

Comment:
This would actually be great in the base XCache image. Could you add this [here](https://github.com/opensciencegrid/docker-xcache/tree/master/xcache/sbin) and fixup the XCache Dockerfile CMD appropriately?